packages/manager/apps/container/src/container/legacy/server-sidebar/universe/public-cloud/useProjects.tsx

```diff
@@ -20,6 +20,7 @@ export function useProjects() {
         'X-Pagination-Mode': 'CachedObjectList-Cursor',
         'X-Pagination-Sort': 'description',
         'X-Pagination-Sort-Order': 'ASC',
+        'Pragma': 'no-cache',
```

Comment:
```suggestion
        'Cache-Control': 'no-cache'
```

Pragma is deprecated try to use Cache-Control and see if its working as well